src/OrbitProducer/CMakeLists.txt

```diff
@@ -22,9 +22,9 @@ target_include_directories(OrbitProducer PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries(OrbitProducer PUBLIC
         GrpcProtos
         OrbitBase
-        OrbitServiceLib
         concurrentqueue::concurrentqueue
-        CONAN_PKG::abseil)
+        CONAN_PKG::abseil
+        ServiceLib)
```

Comment:
What's the order here? I feel like the previous position was fine, in order with our modules and before the external ones.

Reply:
trying to keep it alphabetical... but they were not sorted before...

Reply:
restored.